api-gateway/kustomize/api-gateway/overlays/common/oathkeeper/access-rules.tmpl

```diff
@@ -8,7 +8,7 @@
   {
     "id": "issuer-credential-management",
     "upstream": {
-      "url": "http://vcs"
+      "url": "http://vcs-aws-kms"
```

Comment:
Can we rename this to vcs-awskms to be in sync with vcs-localkms?

Reply:
yes sure